webpack.config.js

```diff
@@ -10,14 +10,13 @@ const OptimizeCssAssetsPlugin = require("optimize-css-assets-webpack-plugin");
 const safePostCssParser = require("postcss-safe-parser");
 
 const prodPlugins = (isDev) => {
-  if (isDev) {
-    return [];
-  }
+  // if (isDev) {
+  //   return [];
+  // }
```

Comment:
Can we disable push notification in development for better performance while compiling?